frappe/email/receive.py

```diff
@@ -321,6 +328,46 @@ def process_part(self, part):
 		if part.get_filename():
 			self.get_attachment(part, charset)
 
+	def get_attached_message(self,part):
+		list=[None]*4
+		list[0]=part.__getitem__('from')
+		list[1]=part.__getitem__('to')
+		list[2]=part.__getitem__('subject')
+		list[3]=part.__getitem__('Date')
+
+		if None in list[:3]:#check if date returned if it didnt find a recieved field and get date from there
+			return False
+		if list[3]==None:
+			list[3] = part.__getitem__('Received')
+			temp = list[3].split(';')
+			list[3] = temp[len(temp)-1]
+
+		self.html_content +=  '<hr>'
+		list[0] = list[0][list[0].find("<")+1:list[0].find(">")]#From: cleaner
+
+		temp =""#To: cleaner
+		for d in list[1].split(','):
+			temp += d[d.find("<")+1:d.find(">")] +','
+		list[1] = temp.strip(',')
+
+		self.text_content +=   'From: {0}\nTo: {1}\nSent: {2}\nSubject: {3}\n'.format(list[0],list[1],list[3],list[2])
+		self.html_content +=   '<p>From: {0}</p><p>To: {1}</p><p>Sent: {2}</p><p>Subject: {3}</p><br>'.format(list[0],list[1],list[3],list[2])
```

Comment:
Why do you need to add all this to html and text content? This isn't what's there for other services. Why go for a non-standard display?